### PR TITLE
Adopt skyux-sdk/builder

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,19 +25,64 @@
...
-    "@skyux/grids": "^3.0.0",
-    "@skyux/i18n": "^3.5.0",
-    "@skyux/indicators": "^3.0.0",
-    "@skyux/list-builder": "^3.2.1",
-    "@skyux/list-builder-view-checklist": "^3.0.0",
+    "@skyux/grids": "^3.2.0",
+    "@skyux/i18n": "^3.7.0",
+    "@skyux/indicators": "^3.1.0",
+    "@skyux/list-builder": "^3.5.0",
+    "@skyux/list-builder-view-checklist": "^3.1.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json:32]: Do we need to update the peers?

File: package.json [hunk 2]
```diff
@@ -25,19 +25,64 @@
...
+    "@angular/core": "7.2.15",
+    "@angular/forms": "7.2.15",
+    "@angular/http": "7.2.15",
+    "@angular/platform-browser": "7.2.15",
+    "@angular/platform-browser-dynamic": "7.2.15",
+    "@angular/router": "7.2.15",
+    "@blackbaud/auth-client": "2.17.2",
+    "@pact-foundation/pact": "9.0.1",
+    "@pact-foundation/pact-web": "7.4.0",
+    "@skyux-sdk/builder": "^3.8.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json:52]: Caret `^`

File: package.json (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply package.json:32]: Maybe not in this PR, but I noticed a serious issue - if you install the latest `view-grids`, but only have grids `3.0` installed, you will have an issue when trying to use the `settingsKey` input --> it didn't exist until version `3.1`. Figured I could kill two birds with one stone here. Think that's bad form?

File: package.json (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply package.json:52]: Good eye! Fixed.

File: package.json (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply package.json:32]: Not at all. I'm okay with updating any of the peers as long as it's a requirement.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**